### ISSUE TITLE
EDL * raise EDLParseError

### ISSUE THREAD

**Issue by col-one** [PixarAnimationStudios/OpenTimelineIO#244]
Hello If I try to parse an edl like : 
```
TITLE: TOTO
FCM: NON-DROP FRAME
001  001      V     C        00:00:00:00 00:00:02:01 01:00:00:00 01:00:02:01  
* FROM CLIP NAME: sht0010.mov
002  001      V     C        00:00:00:00 00:00:02:16 01:00:02:01 01:00:04:17  
* FROM CLIP NAME: sht0020.mov
003  001      V     C        00:00:00:00 00:00:02:23 01:00:04:17 01:00:07:16  
* FROM CLIP NAME: sht0030.mov
```
and the cmx_3600 parser raise EDLParseError.

If I add an other exception like to the parse_edl : 

```python
            elif line.startswith('*'):
                # this can occur either in the header or before any 'event'
                # in both cases we can ignore it since it is meant for tape
                # *
                pass
```

Is it a bug ? I missed something ?

**Comment by col-one** [comment]
Precisely, It's the last * comment line who raise the error.

**Comment by col-one** [comment]
Ok there is some empty values in the edl_lines list, so first element edl_lines[0] is not the comment. 
I patched it with : 
```python
                    edl_lines = list(filter(None, edl_lines))
                    if re.match(r'^\D', edl_lines[0]):
                        comments.append(edl_lines.pop(0))
                    else:
                        break
```

**Comment by jminor** [comment]
Which version of OTIO are you using? Can you paste the full exception with the stack trace? When I try the EDL you posted above with the current OTIO, it works fine.

**Comment by ssteinbach** [comment]
Addressed by #255

**Comment by ssteinbach** [comment]
Haven't heard anything more on this issue so closing it for now.  Feel free to re-open if you're still seeing the problem.  Thanks!
